frontend/src/components/CategorySelect/CategorySelect.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC, useState } from 'react';
+
+import { typedMemo } from 'src/core/utils/typed-memo';
+import { Category } from 'src/core/models/category';
+import { Typography } from 'antd';
+
+import './CategorySelect.scss';
+
+const { Text } = Typography;
+
+type Props = Readonly<{
+
+  /** Массив категорий навыков. */
+  categories: Category[];
+
+  /** Ф-ция изменения выбранных категорий. */
+  onChange: (selectedIds: number[]) => void;
+}>;
+
+// Компонент Панель выбора категорий навыков (фильтр по категориям)
+const CategorySelectComponent: FC<Props> = (props: Props) => {
```

Comment:
Generic у `FC` автоматически задаёт тип пропсам. Ты можешь опустить значение в функции не сломав тип

```suggestion
const CategorySelectComponent: FC<Props> = (props) => {
```